src/pages/products/View.tsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import React, { useEffect, useState } from 'react';
+import { ProductDetails } from 'common/interfaces/ProductDetail';
+import { Card, Element } from '@invoiceninja/cards';
+import { useTranslation } from 'react-i18next';
+import { generatePath, useParams } from 'react-router';
+import { useProductQuery } from '../../common/queries/products';
+import { Container } from '../../components/Container';
+import { Default } from '../../components/layouts/Default';
+import { Spinner } from '../../components/Spinner';
+import { Breadcrumbs } from 'components/Breadcrumbs';
+import { Tab, Tabs } from '../../components/Tabs';
+
+export function View() {
+  const [t] = useTranslation();
+  const { id } = useParams();
+  const [ProductDetails, setProductDetails] = useState<ProductDetails>(
```

Comment:
This variable is out of the convention of the project. Please use camelCase for variables.